code/modules/shuttle/cas_plane.dm

```diff
@@ -268,6 +268,7 @@
 	if(eyeobj.eye_user)
 		to_chat(user, span_warning("CAS mode is already in-use!"))
 		return
+	SSmonitor.process_human_positions()
 	if(SSmonitor.human_on_ground <= 5)
```

Comment:
can i spam the button to lag the server or how expensive is this

Reply:
not expensive